src/CodeGenerator/src/Generator/ResponseParser/RestJsonParser.php

```diff
@@ -267,9 +267,9 @@ private function parseResponseMap(MapShape $shape, string $input, bool $required
                     $this->functions[$keyName] = strtr($body, [
                         'FUNCTION_KEY' => \var_export($keyName, true),
                         'CLASS' => $shape->getValue()->getShape()->getName(),
-                    ]);
+                    ]);// TODO add CLASS to imports
```

Comment:
How can I do this in a good way?

Reply:
I hesitate between 
- returning the list of classes used (and to register)
- passing the namespace to the method to let the parser/dumper register the class by themselves
- creating a ClassUsageService injected in all code generators with 3 methods `start($namespace)`, `end()`, `use(ClassName $class)`

Reply:
There is a forth option: Just add all value objects as `use` statements and let cs-fixer remove them. 

Im not sure what the best way forward is. I'll explore your options first.

Reply:
I returned a list of classes. https://github.com/async-aws/aws/pull/677/commits/55dba4105f55b39a7f9f3843b36b2daeab819a98